.gitignore

```diff
@@ -1,3 +1,6 @@
 /oci-create-runtime-bundle
 /oci-unpack
 /oci-image-validate
+
+*.1
+*.patch
```

Comment:
we always work with patch, and easy to forget to remove them before adding. I think it'd better to ignore them.